float32 latitude
float32 longitude
float32 altitude
int16 satellites
